model/src/main/kotlin/OrtResult.kt

```diff
@@ -403,10 +404,30 @@ data class OrtResult(
     fun getAdvisorResultsForId(id: Identifier): List<AdvisorResult> = advisorResultsById[id].orEmpty()
 
     /**
-     * Return all [RuleViolation]s contained in this [OrtResult].
+     * Return all [RuleViolation]s contained in this [OrtResult]. Optionally exclude resolved violations with
+     * [omitResolved] and remove violations below the [minSeverity].
      */
     @JsonIgnore
-    fun getRuleViolations(): List<RuleViolation> = evaluator?.violations.orEmpty()
+    fun getRuleViolations(omitResolved: Boolean = false, minSeverity: Severity? = null): List<RuleViolation> {
+        val allViolations = evaluator?.violations.orEmpty()
+
+        val violationsWithMinSeverity = when (minSeverity) {
```

Comment:
Nit: `violationsWithMinSeverity` is not strictly correct, but `violationsWithAtLeastMinSeverity` is very long, so maybe just `severeViolations`?